test/shared.py

```diff
@@ -30,11 +30,7 @@
 EXPECTED_CLASS_HASH = (
     "0x18272236f94c858935851a7024eff968eea3af2b8ed2ad02c16917068f23866"
 )
-
-# This is temporary change.
-EXPECTED_SALTY_DEPLOY_HASH_LITE_MODE = (
-    "0x342d5cd9a280b81112518dc7afd9ae10e0ce484389aa0c2f626727498a51594"
-)
+EXPECTED_SALTY_DEPLOY_HASH_LITE_MODE = "0x2"
```

Comment:
If these two constants are used in just one file, it makes more sense for them to be defined there than in this file called `shared.py`

Reply:
true, updated